app/containers/Sessions/containers/SettingsSessionPage/components/SessionSettings.js

```diff
@@ -204,6 +222,37 @@ const SessionSettings = ({
                 />
               </InputContainer>
 
+              {type === SessionTypes.SMS_SESSION && (
+                <InputContainer style={{ marginTop: 15 }}>
```

Comment:
Add margin style props to InputContainer:
```
export const InputContainer = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  ${margin}; // <---------
`;
```

Then you'll be able to use `mt` and `mb` props. Update other usages too